CHANGES.md

```diff
@@ -9,6 +9,9 @@
 
 * `augur.io.read_metadata`: A new optional `dtype` argument allows custom data types for all columns. Automatic type inference still happens by default, so this is not a breaking change. [#1252][] (@victorlin)
 * `augur.io.read_vcf` has been removed and usage replaced with TreeTime's function of the same name which has improved validation of the VCF file. [#1366][] (@jameshadfield)
+* `augur parse`: A new optional `--output-id-field` argument allows the user to select any ID field for the produced FASTA file (e.g. 'accession' instead of 'name' or 'strain'). [#1403][] (@j23414)
+  * When no `--output-id-field` is given and the data has both `name` and `strain` fields, continue to preferentially use `name` over `strain` as the sequence ID field; but, throw a deprecation warning that the order will be switched to prefer `strain` over `name` in the future to be consistent with the rest of Augur.
+  * Added entry to [DEPRECATED.md](./DEPRECATED.md).
```

Comment:
This was added to the section for 24.1.0. You'll want to move it to the section for `__NEXT__`. Might be best to rebase onto latest `master` since I've merged a few PRs that also updated the changelog.

Reply:
Thanks for catching this! I completely misread the headers 😅 

Hopefully https://github.com/nextstrain/augur/pull/1403/commits/d313f580c69386df83191cd9aedb95fca39cb080 adds the Changelog message in the right place. I also went ahead and rebased and fixup'd all the commits.